src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapter.java

```diff
@@ -1,37 +1,38 @@
-package edu.kis.powp.jobs2d.drivers.adapter;
-
-import edu.kis.legacy.drawer.panel.DrawPanelController;
-import edu.kis.legacy.drawer.shape.ILine;
-import edu.kis.legacy.drawer.shape.LineFactory;
-import edu.kis.powp.jobs2d.Job2dDriver;
-
-/**
- * driver adapter to drawer with several bugs.
- */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
-	private int startX = 0, startY = 0;
-
-	public MyAdapter() {
-		super();
-	}
-
-	@Override
-	public void setPosition(int x, int y) {
-		this.startX = x;
-		this.startY = y;
-	}
-
-	@Override
-	public void operateTo(int x, int y) {
-		ILine line = LineFactory.getBasicLine();
-		line.setStartCoordinates(this.startX, this.startY);
-		line.setEndCoordinates(x, y);
-		drawLine(line);
-		this.setPosition(x, y);
-	}
-
-	@Override
-	public String toString() {
-		return "@Q!$!@$!#@$(*#@&Q(%^*#@";
-	}
-}
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class DrawerAdapter extends DrawPanelController implements Job2dDriver {
+	private int startX = 0, startY = 0;
+
+	public DrawerAdapter() {
+		super();
+	}
+
+	@Override
+	public void setPosition(int x, int y) {
+		this.startX = x;
+		this.startY = y;
+	}
+
+	@Override
+	public void operateTo(int x, int y) {
+		ILine line = LineFactory.getBasicLine();
+		line.setStartCoordinates(this.startX, this.startY);
+		line.setEndCoordinates(x, y);
+		DrawerFeature.getDrawerController().drawLine(line);

```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.